proj/src/main/webapp/script.js

```diff
@@ -252,12 +252,6 @@ function init() {
     const chat = document.getElementById('chatbox');
     chat.addEventListener('scroll', addMoreMessagesAtTheTop);
 
-    const close = document.getElementById('close-settings');
-    close.addEventListener('click', closeSettings);
-
-    const settings = document.getElementById('settings-button');
-    settings.addEventListener('click', switchToSettings);
-
```

Comment:
There isn't a problem with the actual code; it's just that those functions are not needed anymore because the closing setting is replaced by collapsing the sidebar and the switching to setting is replaced by the settings drop down menu